config/development/config_split.config_split.development.yml

```
@@ -0,0 +1,28 @@
+uuid: fa892fa6-9327-447e-a29c-78170be8198b
+langcode: en
+status: true
+dependencies: {  }
+id: development
+label: development
+description: ''
+weight: 0
+folder: ../config/development
```

It's a good practice if we add split folder within config while naming the folder 
```suggestion
folder: ../config/split/development
```